snorkel/learning/pytorch/rnn/lstm.py

```diff
@@ -55,4 +62,7 @@ def initialize_hidden_state(self, batch_size):
         return (
             torch.zeros(self.num_layers * self.num_directions, batch_size, self.hidden_dim),
             torch.zeros(self.num_layers * self.num_directions, batch_size, self.hidden_dim)
+        ) if self.gpu else (
```

Comment:
@danich1 seems like this condition might be the wrong way around?  I.e. `.cuda()` should happen if self.gpu is true, not the reverse (which is how it seems to be written now)?

@ajratner do you want to put in CI tests for GPU functionality before merging or no?

Reply:
Ack. Good catch. Changing that now.